tremor-pipeline/src/op/trickle/select.rs

```diff
@@ -69,11 +69,27 @@ impl Select {
             .collect();
         let select = srs::Select::try_new_from_stmt(stmt)?;
         let event_id_gen = EventIdGenerator::new(operator_uid);
-        if let ast::Stmt::Select(SelectStmt { aggregates, .. }) = stmt.suffix() {
+        if let ast::Stmt::Select(SelectStmt {
+            stmt: _,
+            aggregates,
+            consts,
+            locals: _,
```

Comment:
using `, ..` is usually a better choice as it makes refactoring easier (if we add a field we don't need to change every occurrence where it's not used)